lib/molecules/logo/directive.js

```diff
@@ -0,0 +1,16 @@
+angular
+  .module('lnPatterns')
+  .directive('lnLogo', lnLogo);
+
+function lnLogo() {
+
+  return {
+    restrict: 'A',
```

Comment:
@nacho87 @mitogh 

To implement it as an element we should use a component instead of a directive: https://docs.angularjs.org/guide/component

If you really need a directive, we should implement it as an attribute. This is an angular recommendation to facilitate things for a future angular2 migration.